packages/cli/src/ActiveWorkflowRunner.ts

```diff
@@ -58,21 +60,27 @@ export class ActiveWorkflowRunner {
 
 		this.activeWorkflows = new ActiveWorkflows();
 
+		const logger = LoggerProxy.getInstance();
+
 		if (workflowsData.length !== 0) {
-			console.log('\n ================================');
-			console.log('   Start Active Workflows:');
-			console.log(' ================================');
+			logger.info('\n ================================');
+			logger.info('   Start Active Workflows:');
+			logger.info(' ================================');
```

Comment:
When do we call the logger as `LoggerProxy.getInstance();` (see above) and when as `getLogger` (originally `getInstance`)? The interfaces are identical.

Reply:
Hello @ivov 

You should usually only use `LoggerProxy.getInstance()` as it should be available project-wide.

The `getLogger` function is used to initialize the logger for the first time and it is then provided to the `LoggerProxy`.

In a few cases, when the LoggerProxy has not been initialized yet you can use `getLogger` directly but it is recommended avoiding it as much as possible.

If it's unclear please let me know and I can clarify.